index.bs

```diff
@@ -427,6 +427,55 @@ spec:reporting; urlPrefix: https://w3c.github.io/reporting/
       </div>
     </section>
   </section>
+  <section>
+    <h3 id="meta-element">
+      The `<meta>` element
+    </h3>
+
+    A {{Document}} may deliver a policy via one or more HTML <{meta}> elements
+    whose <{meta/http-equiv}> attributes are an <a>ASCII case-insensitive</a>
+    match for the string "`Feature-Policy`". For example:
+
+    <div class="example">
+      <pre highlight="html">
+        &lt;meta http-equiv="Feature-Policy" content="fullscreen 'none'; geolocation 'none'"&gt;
+      </pre>
+    </div>
+
+    Implementation details can be found in HTML's <a>Feature Policy state</a>
+    `http-equiv` processing instructions [[!HTML]].
+
+    Authors are <em>strongly encouraged</em> to place <{meta}> elements as early
+    in the document as possible, because policies in <{meta}> elements are not
+    applied to content which precedes them.
+
+    <div class="example">
+
+      For example, a script tag that preceded a &lt;meta&gt;-delivered policy
+      directive of `sync-xhr ‘none’` would still be able to call
+      `XMLHttpRequest.open()`.
+
+      <pre highlight="html">
+        &lt;script&gt;
+          var req = new XMLHttpRequest();
+          req.open("GET", "/api/security_check.json", false);
+          req.send();
+        &lt;/script&gt;
+
+        &lt;meta http-equiv="Feature-Policy" content="sync-xhr ‘none"&gt;
+      </pre>
+
+      In the above, the call to req.open will succeed, because the policy
+      disabling sync-xhr occurs after it, in the markup. 
+
+    </div>
+
+    Note: A policy specified via a <{meta}> element will be enforced along with any other policies active, regardless of where they're specified. However, once a specific Feature Policy directive has been set by way of an HTTP header or <{meta}> element, that directive cannot be further updated or changed.
+    <!--TODO this is first wins. do we want last wins? -->
```

Comment:
How multiple `Feature-Policy` headers should be combined is the subject of #148. I don't think this PR should change anything other than adding text to say that using `<meta>` for Feature Policy is allowed, and each such `<meta>` is interpreted as though a separate `Feature-Policy` header field were appended to the HTTP response header. Then the text that resolves #148 can explain how multiple 'Feature-Policy' header fields are combined together.

Reply:
I don't think that's possible as they are parsed independently. If you start a quoted string in one and end inside a `<meta>` that should not work.

Reply:
Good point :)

A `<meta>` element would have to be parsed and merged separately from the response headers. 

That's also necessary because it would be encountered after [create a new browsing context](https://html.spec.whatwg.org/multipage/browsers.html#creating-a-new-browsing-context) has already initialized the document's policy (unless we specify that it interacts with the preload scanner somehow, but I think there is legitimate hesitancy to adding more dependencies on that).

Reply:
> > and each such `<meta>` is interpreted as though a separate `Feature-Policy` header field were appended to the HTTP response header.
>
> I don't think that's possible as they are parsed independently. 

I think you're probably interpreting me too literally. I just meant that each `<meta http-equiv=Feature-Policy>` should be interpreted the same as if it were a HTTP header.

More generally, it seems bad that CSP and Feature-Policy as specifying so many details for `<meta http-equiv>`; it seems like HTML5 itself should provide more infrastructure for these specs to reuse.

Reply:
Well, for starters there's disagreement on whether we should add more `<meta http-equiv>`. I don't think it's a good idea as it makes all these policies dynamic and there's usually subtly different processing models across browsers leading to issues.

Then, the value-space is different, headers contain bytes, attribute values contain strings. Then, headers can be parsed as a single-block (and ideally combined, then parsed), elements are typically parsed case-by-case.

There are some reusable bits there, but all of it requires quite a bit of care.

And then, for consistency across `<meta http-equiv>`, it's unclear to what extent that exists today, but if someone were to invest time in that maybe some abstraction is feasible as you suggest.